### ISSUE TITLE
Is there a way to make banner like firebase_admob ?

### ISSUE THREAD

**Issue by pptavozl** [googleads/googleads-mobile-flutter#359]
I had better results with firebase_admob because my app doesn't load a new app every screen that open.

Is there a way to implement top of all bottom banner (like firebase_admob plugin) ?

**Comment by maheshmnj** [comment]
Hi @pptavozl, Thanks for filing the issue, 

> my app doesn't load a new app on every screen that opens. Is there a way to implement top of all bottom banner 

I am not sure I understand your issue could you rephrase it with some examples?
